### PR TITLE
:sparkles: added Bloc intro for newbies

### PR DESCRIPTION
**Pull request by UmairJibran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This section has projects that have just a basic UI and simple logic with locali
 - [Reading Local Json Data](https://github.com/uzairleo/flutter_json-storage-)
 - [Flutter-SplashScreen](https://github.com/uzairleo/flutter_splashscreen_tutorial)
 - [Top10-Flutter-Widget](https://github.com/uzairleo/top10_flutter_widget)
+- [BLoC App](https://github.com/UmairJibran/flutter-bloc.git)
```
Comment (haroonkhan9426) [inline README.md]: Best is to add it into the intermediate section. 
The architectural concepts are not of basic level as it requires some initial knowledge.

Comment (UmairJibran) [reply README.md]: OK will do it right now.

**Review by haroonkhan9426 — COMMENTED** [review]
Now good to go.

**Comment by UmairJibran** [comment]
Thanks Appretiated.

**Pull request merged**